ts-client/src/dlmm/index.ts

```diff
@@ -4284,6 +4286,168 @@ export class DLMM {
     };
   }
 
+  /**
+ * The `seedLiquidity` function create multiple grouped instructions. The grouped instructions will be either [initialize bin array + initialize position instructions] or [deposit instruction] combination.
+ * @param
+ *    - `owner`: The public key of the positions owner.
+ *    - `seedAmount`: Lamport amount to be seeded to the pool.
+ *    - `price`: Price in UI format
+ *    - `base`: Base key
+ *    - `roundingUp`: 
+ * @returns {Promise<Transaction>}
+ */
+  public async seedLiquiditySingleBin(
+    owner: PublicKey,
+    base: PublicKey,
+    seedAmount: BN,
+    price: number,
+    roundingUp: boolean
+  ): Promise<Transaction> {
+    const toLamportMultiplier = new Decimal(
+      10 ** (this.tokenY.decimal - this.tokenX.decimal)
```

Comment:
what if tokenY.decimal <= tokenX.decimal ?

Reply:
the price will be scaled up or down depend on the `(tokenY.decimal - tokenX.decimal)`

Reply:
eg: 1 SOL =  230.52 USDC => input price = 230.52 
Convert to price for 1 lamport = 230.52 * 10 ** (6 -9) = 0.23052 (USDC has 6 decimals and SOL has 9 decimals)